sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/entities/ShareDataBuilder.java

```diff
@@ -127,21 +162,39 @@ public ShareRequestPreviews buildShareDataPreview(ShareRequestInfo requestInfo)
 		List<SormasToSormasEventPreview> events = new ArrayList<>();
 		List<SormasToSormasEventParticipantPreview> eventParticipants = new ArrayList<>();
 
+		List<ValidationErrors> validationErrors = new ArrayList<>();
```

Comment:
`valiadtionErrors` are never checked and the preview request is sent empty

Reply:
fixed